change/@azure-msal-angular-2020-08-11-10-47-44-redirect-docs.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Comment:
This should be type none, right?

Reply:
I considered that, should it be? Honestly not sure how we want to use these tags. I went with "patch" since it was addressing a problem from GitHub even though this is only documentation.

Reply:
The type determines how the version will be bumped. Since this doesn't touch the library, this shouldn't bump the version

I believe the types correspond to version bumps of:
<major>.<minor>.<patch>-<Prerelease??>

Reply:
Makes sense, fixed it now.